modules/virt-example-inherit-static-ip-from-nic.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * virt/node_network/virt-updating-node-network-config.adoc
+
+[id="virt-example-inherit-static-ip-from-nic_{context}"]
+= Example: Linux bridge interface node network configuration policy to inherit static IP address from the NIC attached to the bridge
+
+Create a Linux bridge interface on nodes in the cluster and transfer the static IP configuration of the NIC to the bridge by applying a single `NodeNetworkConfigurationPolicy` manifest to the cluster.
+
+The following YAML file is an example of a manifest for a Linux bridge interface. It includes samples values that you must replace with your own information.
+
+
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: <br1-eth1-copy-ipv4-policy> <1>
+spec:
+  nodeSelector: <2>
+    node-role.kubernetes.io/worker: "" <3>
+  capture:
+    eth1-nic: interfaces.name=="eth1"
```

Comment:
Can we reference the expressions syntax from here ? https://nmstate.io/nmpolicy/user-guide/102-policy-syntax.html

Reply:
I think this should be <4>, the capture worth an explanation.

Reply:
Direct preview link: https://deploy-preview-40886--osdocs.netlify.app/openshift-enterprise/latest/virt/node_network/virt-updating-node-network-config.html#capturing-nic-static-ip_virt-updating-node-network-config

Reply:
> @qinqon Are you saying that we should add a link to the policy syntax page in the docs? 

Is that possible ? or we cannot reference the kubernetes-nmstate docs ?

Reply:
yes, it would be nice to have the policy syntax. With the suggested example only, I"m not sure the user will understand how the `capture` works.